index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+<body>  
+    <div class="screen">
+        <header class="header">
+            <div class="time-status">
+                <div class="time">04:20 am</div>
+                <div class="status">
+                   <img src="images/status.svg" alt="status" class="status"> 
+                </div>
+            </div>
+            <div class="header-items">
+                <div class="my-logo">
+                    <a class="logo-text">
+                        John Doe
+                    </a>
```

Comment:
### Before removing width
![before](https://user-images.githubusercontent.com/93039359/185612586-2566cc70-9121-4934-8067-cec63a0a664d.png)
### After removing width
![Screenshot from 2022-08-19 16-52-37](https://user-images.githubusercontent.com/93039359/185612705-f73d125a-5ace-42c7-8079-3376c28b98c4.png)


- [ ] Please remove width as shown in the image below to make your logo look like Figma :+1:
**Quick Fix**
![Screenshot from 2022-08-19 16-50-49](https://user-images.githubusercontent.com/93039359/185612809-bcf681f8-e4ee-445e-9789-e358590c6845.png)